Documentation/tectonic.md

```diff
@@ -0,0 +1,56 @@
+# Using Rook on Tectonic Bare Metal
+
+Here is a running guide on how to implement Rook on Tectonic. A complete guide on how to install Tectonic is out of the scope of the Rook project. But many info can be found on the [Tectonic website](https://coreos.com/tectonic/docs/latest/)
+
+## Prequisites
+
+- An installed tectonic-installer. These steps are described on [the Tectonic website](https://coreos.com/tectonic/docs/latest/install/bare-metal/#4-tectonic-installer)
+- A running matchbox node which will do the provisioning (Matchbox is only required if you are running Tectonic on Bare metal)
+- You can run through all steps of the GUI installer, but in the last step, choose `Boot manually`. This way we can make the necessary changes first.
+
+## Edit the kubelet.service file to be compatible with Rook
+
+To make the kubelet compatible with Rook we need to make a few adaptions to the systemd service file generated by the Tectonic-installer.
```

Comment:
I think `compatible` isn't the correct word here. It is more of adding a flag so that the default which is `/usr/libexec/...` is set to `/var/lib/...`, a path reachable outside of the Kubelet container.

Reply:
I understand and couldn't find the right word, so I deleted the word `compatible` in the first sentence and in the text.